tests/common_test.py

```diff
@@ -0,0 +1,91 @@
+import os
+import pytest
+from testbook import testbook
+import json
+from pytest_dependency import depends  # import the depends function
+
+
+def auth() -> tuple[str, str]:
+    cred_file_path = f"{os.path.expanduser('~')}/.gcp/doc-test-creds.json"
+    assert os.path.isfile(cred_file_path)
+    with open(cred_file_path, "r") as f:
+        creds_json = json.loads(f.read())
+        project_id = creds_json["project_id"]
+    return (
+        f"!gcloud auth activate-service-account --key-file={cred_file_path} -q",
+        f"!gcloud config set project {project_id} -q",
+    )
+
+
+@pytest.fixture(scope="session")
+def tb_quickstart():
+    with testbook("docs/quickstart.ipynb", execute=False, timeout=1800) as tb:
+        yield tb
+
+
+@pytest.fixture(scope="session")
```

Comment:
I think all of the auth_tb functions can be removed if we use default credentials

Reply:
Each of those fixtures are specific to a single notebook file so I couldn't find a way around duplicating them. I tried to collapse them into a single parameterized fixture but I quickly realized that wasn't actually what I was after. (https://docs.pytest.org/en/7.3.x/how-to/fixtures.html#fixture-parametrize)